Feedback on chain component

diegoalzate opened the pull request:
closes #354 

### overview
created a `FeedbackTransaction` that receives 

```
{
  transactionHash?: Address;
  confirmations: number;
  feedbackTexts: {
    loading: string;
    idle?: string;
    success?: string;
    error?: string;
  };
}
```

Depending on the status of the transaction hash it will show the feedback text. Confirmation number allows customization on how long we would want to wait to make sure a tx went through.

### ui
![CleanShot 2023-09-07 at 16 20 29](https://github.com/hoprnet/hopr-admin/assets/22416585/bcdfe9e1-f131-4eb7-b25c-2f4cb10cf005)

mjadach-iv commented on config.ts:
```diff
@@ -27,8 +27,8 @@ export const STAKING_V2_SUBGRAPH = 'https://stake.hoprnet.org/api/hub/subgraph-d
-export const MINIMUM_WXHOPR_TO_FUND = 30_000;
+export const MINIMUM_WXHOPR_TO_FUND = 0.03;
 export const MINIMUM_WXHOPR_TO_FUND_NFT = 10_000;
-export const MINIMUM_XDAI_TO_FUND = 2;
-export const MINIMUM_XDAI_TO_FUND_NODE = 1;
+export const MINIMUM_XDAI_TO_FUND = 0.02;
+export const MINIMUM_XDAI_TO_FUND_NODE = 0.01;
```
Need to revert that

mjadach-iv reviewed (changes requested):
Please use this component in all places that require wallet confirm

diegoalzate commented:
> Please use this component in all places that require wallet confirm

Added to fund node but did not add it to sign functions because we do not wait on those. Also reverted the minimum values.

mjadach-iv reviewed (changes requested):
- not present on 'staking/withdraw' path
- not present on 'staking/stake-wxHOPR' path
- not present on 'staking/fund-node' path

mjadach-iv reviewed (changes requested):
Still doesn't show on some components, eg:
![image](https://github.com/hoprnet/hopr-admin/assets/104129084/52dbec87-c460-4948-b81b-86fcb89a5a9d)

diegoalzate commented:
> Still doesn't show on some components, eg: ![image](https://user-images.githubusercontent.com/104129084/267886156-52dbec87-c460-4948-b81b-86fcb89a5a9d.png)

added a isWalletPendingState for this use case

mjadach-iv reviewed (changes requested):
Right now the `/staking/withdraw` is jumping when we press 'Execute' because the Feedback component is showing up. It also might be the case on other components.
It would be best to make the component have min-height of its normal height when displayed, so when the status would show, nothing would be moving around on the scren
